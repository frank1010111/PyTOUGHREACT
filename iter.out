 MESH: NEL=          12  NCON=          17
 TOUGH STATUS: Preprocessing step completed.
 TOUGH STATUS: Total preprocessing time =   0.000000000000000E+000
