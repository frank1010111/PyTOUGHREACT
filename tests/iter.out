                     TOUGHREACT V3.60demo

          ---- Modelling Progress and Iteration Messages ----

 Description of column-headers and variables:
   TIMESTEP = Total number of timesteps
   TIME     = Total simulation time (seconds and years)
   ITERFL   = Iterations to solve flow
   ITERSQ   = Sequential transport-chemistry iterations
   MAXITCH  = Maximum iterations to solve chemistry
   AVGITCH  = Average iterations to solve chemistry
   DELTA-T  = Current time step (seconds)
   DT LIMIT = Criterion limiting timestep
   MAXITCH ID = Grid block: maximum chemical iterations

 TIMESTEP    TIME(sec)     TIME(yr)  ITERFL ITERSQ MAXITCH AVGITCH  DELTA-T(sec)  DT LIMIT     MAXITCH ID
 ----------------------------------------------------------------------------------------------------------
        0  0.000000E+00  0.000000E+00    0    0      5      5.000   0.00000E+00      ---           InWat
        1  8.640000E+03  2.737912E-04    2    1      9      9.000   8.64000E+03      ---             a 1
        2  1.728000E+04  5.475823E-04    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
        3  2.592000E+04  8.213735E-04    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
        4  3.456000E+04  1.095165E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
        5  4.320000E+04  1.368956E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
        6  5.184000E+04  1.642747E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
        7  6.048000E+04  1.916538E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
        8  6.912000E+04  2.190329E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
        9  7.776000E+04  2.464120E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       10  8.640000E+04  2.737912E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       11  9.504000E+04  3.011703E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       12  1.036800E+05  3.285494E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       13  1.123200E+05  3.559285E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       14  1.209600E+05  3.833076E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       15  1.296000E+05  4.106867E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       16  1.382400E+05  4.380658E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       17  1.468800E+05  4.654450E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       18  1.555200E+05  4.928241E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       19  1.641600E+05  5.202032E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
       20  1.728000E+05  5.475823E-03    2    1      9      9.000   8.64000E+03  Flo_Delta_t         a 1
     1000  8.640000E+06  2.737912E-01    2    1     10     10.000   8.64000E+03  Flo_Delta_t         a 1
     2000  1.728000E+07  5.475823E-01    2    1     10     10.000   8.64000E+03  Flo_Delta_t         a 1
     3000  2.592000E+07  8.213735E-01    2    1     10     10.000   8.64000E+03  Flo_Delta_t         a 1
     3651  3.154000E+07  9.994645E-01    2    1     10      9.000   4.00000E+03  Tot_time_Dt         a 1
 
        Total number of chemical iterations =      437280
        TOUGHREACT V3.60demo  simulation finished
